start_integration.py

```diff
@@ -1,11 +1,16 @@
+import sys
+import subprocess
+
+subprocess.check_call([sys.executable, '-m', 'pip', 'install', '-r', 'python_dependencies.txt'])
+
 import json
 
 from sms_notifservice import SmsNotifService
 
 with open('settings.json', "rb") as SFile:
     pwd_data = json.loads(SFile.read().decode('utf-8'))
 
-ov_url = "https://" + pwd_data["oneVizionUrl"]
+ov_url = pwd_data["oneVizionUrl"]
```

Comment:
What will happen if customers has integration where url without protocol part specified in the settings file?

Reply:
ok
I've added 2 functions for adding and removing "https://" part so now we can use URL with or without "https://" in the settings file.